src/XRP/issued-currency-client.ts

```diff
@@ -453,6 +453,32 @@ export default class IssuedCurrencyClient {
     )
   }
 
+  /**
+   * Freezes the trust line between this account (issuing account) and another account.
+   * Note that the trust line's limit is set to 0.
+   *
+   * @see https://xrpl.org/freezes.html#enabling-or-disabling-individual-freeze
+   *
+   * @param accountToFreeze The X-Address of the account which to to freeze a trust line.
+   * @param currencyName The currency of the trust line to freeze.
+   * @param wallet The wallet freezing the trust line.
+   */
+  public async freezeTrustLine(
+    accountToFreeze: string,
```

Comment:
I think that in keeping with the rest of the SDK, we need to check that this is an X-Address and throw if it's not?  We should consider as a group whether it makes sense to ALWAYS require X-Addresses everywhere even in situations where there can't possibly BE a tag... in which case we might want to support both address types.